Xamarin.Forms.Core/Shell/ShellContent.cs

```diff
@@ -74,11 +74,6 @@ Page IShellContentController.GetOrCreateContent()
 
 		void IShellContentController.RecyclePage(Page page)
 		{
-			if (ContentCache == page)
```

Comment:
do we even need the recycle call anymore? iOS doesn't use this call and it's kind of confusing